Sources/Basic/Process.swift

```diff
@@ -293,6 +293,15 @@ public final class Process: ObjectIdentifierProtocol {
         var outputPipe: [Int32] = [0, 0]
         var stderrPipe: [Int32] = [0, 0]
         if redirectOutput {
+         
+            #if os(macOS)
+                setlinebuf(Darwin.stdout)
+                setlinebuf(Darwin.stderr)
+            #else
+                setlinebuf(Glibc.stdout)
+                setlinebuf(Glibc.stderr)
+            #endif
```

Comment:
Hmm, I am not sure if this is correct. I don't know these APIs to comment properly. Can you write a test case first? The test should fail initially and start passing after applying the fix. That makes it a little easier to reason about a patch!

Reply:
cool 👍

Reply:
I will test in on Ubuntu as well I have 17.10 in my /pc,i noticed ci run tests on 16.04 should i download image for 16.04 or 17.10 will work fine ?

Reply:
16.04 will be better

Reply:
thanks 😊